src/Avalonia.Dialogs/ManagedFileChooserViewModel.cs

```diff
@@ -17,6 +17,7 @@ internal class ManagedFileChooserViewModel : InternalViewModelBase
         private readonly ManagedFileDialogOptions _options;
         public event Action CancelRequested;
         public event Action<string[]> CompleteRequested;
+        public event Action<string> ShowOverwritePrompt;
```

Comment:
This then would simply be `OverwritePrompt`

Reply:
Addressed